iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/facade/MDStoreFacadeFactory.java

```diff
@@ -0,0 +1,17 @@
+package eu.dnetlib.iis.wf.export.actionmanager.entity.facade;
+
+import java.util.Map;
+
+/**
+ * MDStore service facade factory.
+ * 
+ * @author mhorst
+ *
+ */
+public interface MDStoreFacadeFactory {
+
+    /**
+     * Instantiates {@link MDStoreFacade} for given parameters.
+     */
+    MDStoreFacade instantiate(Map<String, String> parameters);
```

Comment:
I would suggest to name this method 'create' or 'createMDStoreFacade' to comply with the standard naming for the factory design pattern.